metecho/api/models.py

```diff
@@ -950,8 +954,20 @@ def user_logged_in_handler(sender, *, user, **kwargs):
 
 
 def ensure_slug_handler(sender, *, created, instance, **kwargs):
+    slug_field_name = getattr(instance, "slug_field_name", "name")
     if created:
         instance.ensure_slug()
+    elif instance.tracker.has_changed(slug_field_name):
+        # Create new slug off new name:
+        sluggable_name = getattr(instance, slug_field_name)
+        slug = slugify(sluggable_name)
+        slug = instance._find_unique_slug(slug)
+        instance.slug_class.objects.filter(parent=instance.slug_parent).update(
+            is_active=False
+        )
```

Comment:
@wlonk Do we want to be setting `is_active=False`? I think that removes the slug from `old_slugs` entirely, which means it's no longer known to the front end. I assume we want to continue tracking old slugs, so we can redirect to the latest.